src/main/java/com/okta/sdk/clients/AuthApiClient.java

```diff
@@ -114,6 +115,19 @@ public AuthResult authenticate(String username, String password, String relaySta
     }
 
     /**
+     * Authenticate into organization.
+     *
+     * @param  token {@link String}                 activationToken to use for authentication.
+     * @return {@link AuthResult}                   Result of the authentication transaction.
+     * @throws IOException                          If an input or output exception occurred.
+     */
+    public AuthResult authenticate(String token) throws IOException {
+        Map<String, Object> params = new HashMap<String, Object>();
```

Comment:
Should be able to change this to `Map<String, Object> params = new HashMap<>();`.

http://www.javaworld.com/article/2074080/core-java/jdk-7--the-diamond-operator.html